src/Dofs/sparsity_pattern.jl

```diff
@@ -233,3 +287,41 @@ function _condense_sparsity_pattern!(K::SparseMatrixCSC{T}, dofcoefficients::Vec
 
     return nothing
 end
+
+for (func,                              pre_f,                                                                                      inner_f,                                                                                                                                                                    return_values) in (
+    (:cross_element_coupling_count,     :(max_buffer_length = 0;),                                                                  :(max_buffer_length += coupling_length),                                                                                                                                    :(max_buffer_length)),
+    (:cross_element_coupling,           :(I = Vector{Int}(undef, max_buffer_length); J = Vector{Int}(undef, max_buffer_length)),    :(I[cnt+1:cnt+coupling_length] = repeat(cell_dofs,length(neighbour_unique_dofs)); J[cnt+1:cnt+coupling_length] = repeat(neighbour_unique_dofs,inner=length(cell_dofs))),    :(I, J)),
+)
+    @eval begin
+        function $(func)(dh::AbstractDofHandler, topology::ExclusiveTopology, full_cross_element::Bool; max_buffer_length::Int = 0)
+            $(pre_f)
+            element_dof_start = 0
+            cnt = 0
+            for (fhi, fh) in pairs(dh.fieldhandlers)
+                
+                for fi in fh.field_interpolations
+                    # if(!(typeof(fi)<:DiscontinuousLagrange))
+                    #     element_dof_start += getnbasefunctions(fi)
+                    #     continue
+                    # end
```

Comment:
Commenting this makes mixed interpolation symmetric, at least for full coupling for now,
```julia
julia> grid = generate_grid(Line, (3,));

julia> topology = ExclusiveTopology(grid);

julia> ip = DiscontinuousLagrange{1, RefCube, 2}();

julia> ipc = Lagrange{1, RefCube, 1}();

julia> dh = DofHandler(grid);

julia> add!(dh, :u, 1,ip);

julia> add!(dh, :v, 1,ipc);

julia> close!(dh);;

julia> K = create_sparsity_pattern(dh, topology = topology, cross_element_full_coupling = true)
13×13 SparseMatrixCSC{Float64, Int64} with 137 stored entries:
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   ⋅    ⋅    ⋅    ⋅
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   ⋅    ⋅    ⋅    ⋅
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   ⋅    ⋅    ⋅    ⋅
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   ⋅    ⋅    ⋅    ⋅
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
 0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
  ⋅    ⋅    ⋅    ⋅   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
  ⋅    ⋅    ⋅    ⋅   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
  ⋅    ⋅    ⋅    ⋅   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
  ⋅    ⋅    ⋅    ⋅   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0
```